### PR TITLE
Remove map templates from android

### PR DESCRIPTION
**Pull request by juliarei**

### CODE REVIEW

File: android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPAliases.kt [hunk 1]
```diff
@@ -23,7 +23,6 @@ typealias CPListItem = Row
 typealias CPListTemplate = ListTemplate
```
Comment (hgadalrab) [inline android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPAliases.kt:23]: is this file needed if the package is only supporting CarPlay and not Android Auto?

File: android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPEnums.kt [hunk 2]
```diff
@@ -19,9 +19,6 @@ enum class FCPChannelTypes {
     /// Event to update the content of a list template in Android Auto.
```
Comment (hgadalrab) [inline android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPEnums.kt:19]: same here

**Review by hgadalrab — APPROVED** [review]

File: android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPAliases.kt (hunk 1 above)
Comment (juliarei) [reply android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPAliases.kt:23]: Need to remove map related things because of the here_sdk imports. Did not get it running with the Map templates.

File: android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPEnums.kt (hunk 2 above)
Comment (juliarei) [reply android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPEnums.kt:19]: same as above

**Review by hgadalrab — APPROVED** [review]
